.github/workflows/phpunit.yaml

```diff
@@ -0,0 +1,34 @@
+name: Unit tests
+
+on:
+  pull_request:
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Validate composer.json and composer.lock
+      run: composer validate --strict
+
+    - name: Cache Composer packages
+      id: composer-cache
+      uses: actions/cache@v3
+      with:
+        path: vendor
+        key: ${{ runner.os }}-php-${{ hashFiles('**/composer.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-php-
+    - name: Install dependencies
+      run: composer install --prefer-dist --no-progress
+
+    - name: Run test suite
+      env: 
+          IPINFO_TOKEN: ${{ secrets.IPINFO_TOKEN }}
+      run: composer run-script test
```

Comment:
I think we should test on the PHP `7.3`, `7.4`, `8.0` and `8.1` versions in the GitHub actions.